app/src/main/java/edu/uc/groupproject/covid19tracker/ui/main/ByCountryFragment.kt

```diff
@@ -67,14 +66,14 @@ class ByCountryFragment : Fragment() {
             /**
              * Declare custom array adapter and set list view
              */
-            val arrAdapter = ItemAdapter(view.context, android.R.layout.simple_list_item_1, casesListViewItems)
-            countryListView.adapter = arrAdapter
+            val casesListViewAdapter = ItemAdapter(view.context, casesListViewItems)
+            countryListView.adapter = casesListViewAdapter
```

Comment:
Changed variable names to be more "descriptive" https://kotlinlang.org/docs/reference/coding-conventions.html

Reply:
Good.  Avoid variable names like "array", "editText1", "test", "a", etc.  Descriptive variable names mean you can spend less time commenting.